documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -3,38 +3,40 @@
 // // assembly-upgrade.adoc
 
 [id='assembly-upgrade-resources-{context}']
-= Strimzi resource upgrades
+= Strimzi custom resource upgrades
 
-The `v1alpha1` API version is deprecated for the following Strimzi resources:
+[role="_abstract"]
+After you have upgraded Strimzi to {ProductVersion}, you must ensure that your custom resources are using API version `{ApiVersion}`.
 
-* `Kafka`
-* `KafkaConnect`
-* `KafkaConnectS2I`
-* `KafkaConnector`
-* `KafkaMirrorMaker`
-* `KafkaMirrorMaker2`
-* `KafkaTopic`
-* `KafkaUser`
-* `KafkaBridge`
-* `KafkaRebalance`
+IMPORTANT: The upgrade of resources _must_ be performed after xref:assembly-upgrade-{context}[upgrading the Cluster Operator], so the Cluster Operator can understand the resources.
 
-The `v1beta1` API version is deprecated for the following Strimzi resources:
+.CLI upgrades to custom resources
 
-* `Kafka`
-* `KafkaConnect`
-* `KafkaConnectS2I`
-* `KafkaMirrorMaker`
-* `KafkaTopic`
-* `KafkaUser`
+Strimzi provides am _API conversion tool_ with its release artifacts
+From its CLI, you can use the tool to convert the format of your custom resources to `{ApiVersion}`:
```

Comment:
Can we maybe add some more details here? I think it would be good to specify that:
* They can download the ZIP or the TAR.GZ from release artifacts
* They should unpack it and use it using the scripts in the `bin` subdirectory

Reply:
Changed to:

![image](https://user-images.githubusercontent.com/47596553/109682314-3629d280-7b76-11eb-923c-90d792dc4923.png)

I've assumed we'll need a downstream condition, as with our other download section:

ifdef::Downloading[]
You can download its ZIP or TAR.GZ file from {ReleaseDownload}.
endif::Downloading[]
ifndef::Downloading[]
You can download its ZIP or TAR.GZ from {ZipDownload}.
endif::Downloading[]

Reply:
That looks good I think 👍